tests/ui/lifetimes/tail-expr-lock-poisoning.rs

```diff
@@ -3,6 +3,7 @@
 //@ [edition2024] compile-flags: -Zunstable-options
 //@ [edition2024] edition: 2024
 //@ run-pass
+//@ needs-unwind
```

Comment:
I think you can remove the `//@ ignore-wasm` that test directive is only used here due to wasm not supporting panic=unwind.